### PR TITLE
feat(sn-filter-pane): apply collapseMode setting

### PR DESCRIPTION
**Pull request by johanlahti**
## What is changing for the user?
- The new option `collapseMode` can be set to one of these options: `auto`, `always` or `never` to adjust how individual Litboxes should collapse/expand when the rendering space is changing

As long as we don't touch the `collapseMode` option (shown only under flag in sense-client) the behaviour should be exactly as before.

### CODE REVIEW

File: packages/sn-filter-pane/src/components/ListboxGrid/distribute-always-expanded.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// const flatten = (arr = []) => arr.reduce((prev, cur) => prev.concat(cur));
+
```
Comment (veinfors) [inline packages/sn-filter-pane/src/components/ListboxGrid/distribute-always-expanded.ts:2]:
```suggestion
```

File: types/global.d.ts [hunk 2]
```diff
@@ -6,8 +6,16 @@ export interface IListLayout extends EngineAPI.IGenericListLayout {
   },
   layoutOptions?: {
     dense: boolean,
```
Comment (veinfors) [inline types/global.d.ts:8]:
```suggestion
    dense: boolean;
```

File: packages/sn-filter-pane/src/ext/property-panel/data/data-panel/index.ts [hunk 3]
```diff
@@ -117,11 +117,10 @@ export default function getDataPanelItems(env: IEnv) {
     autoConfirm: {
       ref: 'autoConfirm',
       component: 'checkbox',
-      translation: 'properties.filterpane.autoConfirm',
+      translation: 'properties.filterpane.showSelectionsToolbar',
```
Comment (veinfors) [inline packages/sn-filter-pane/src/ext/property-panel/data/data-panel/index.ts:120]: I guess this new string should be added in the sense-client? Do that as soon as possible, so we can get the translations ready soonish

**Comment by veinfors** [comment]
Just left a couple of comments on some mini stuff... But in general - Looks very good!

**Review by veinfors — APPROVED** [review]

File: packages/sn-filter-pane/src/ext/property-panel/data/data-panel/index.ts (hunk 3 above)
Comment (johanlahti) [reply packages/sn-filter-pane/src/ext/property-panel/data/data-panel/index.ts:120]: It's in already (merged on Friday).

**Review by DanielS-Qlik — APPROVED** [review]

**Pull request merged**